readme.md

```diff
@@ -28,35 +28,19 @@ Please make sure you install the following tools before starting with the instal
 
 ## Installation
 
-> Note that you're free to adjust the `~/Sites/pastebin` location to any directory you want on your machine.
+> Note that you're free to adjust the `~/Code/pastebin` location that will be in the `Homestead.yaml` to any directory you want on your machine.
```

Comment:
Can you move this comment inside the `Homestead.yaml.example` file so it's on top and revert your changes to this sentence?